src/hooks/screen/useCreateChat.ts

```diff
@@ -74,6 +75,21 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      setLoadingCreateChat(false);
+      console.log({e}, 'error create chat');
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `createAnonymousChat` function could be improved. Currently, if an error occurs, you're just logging it and setting `setLoadingCreateChat` to false. However, it would be better to also inform the user that something went wrong. Consider adding a toast message or some other form of user-facing error message.
```diff
    } catch (e) {
        setLoadingCreateChat(false);
+       SimpleToast.show('An error occurred while creating the chat');
        console.log({e}, 'error create chat');
    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->